recipe_batches/recipe_batches.py

```diff
@@ -2,13 +2,26 @@
 
 import math
 
+
 def recipe_batches(recipe, ingredients):
-  pass 
+
+    batches = 100000
+
+    for i in recipe:
```

Comment:
you're running two for loops here, making your solution O(n**2). There's ways to make this O(n)